Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,29 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+Its purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### Do
+- access `Current` **only** from topmost `AppDelegate` until `Builder`s layer
+- pass the required dependencies down the chain;
+- create a customized version to fit your specific needs **for tests** and set it in the test `setUp` method
+```swift
+let locale = Locale(identifier: "haw_US")
+Current = World(autoupdatingLocale: locale)
+```
+- reset the instance of `Current` to the default in the `tearDown` method
```

Comment:
```suggestion
```

Reply:
it shouldn't be removed. I prefer the clarity of the separation **do** and **don't**

Reply:
My problem here is that it still feels part of the previous step, as we are talking about how to use current in tests.

Reply:
You're correct. It is indeed part of the previous step. 
This commit (https://github.com/Babylonpartners/ios-playbook/pull/86/commits/27b1f6b10e55e9504ba3d68bb3ae95912dda7b56) I addressed that specific `setUp` and `tearDown` methods changes. 
Thanks.